50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,346 @@
+At least one member should have the ability to run thermal FEA.\@
+
+The software is written in Basic for the PICAXE processor, but will need computer science skillsets to validate the codebase and make any necessary modifications.
+Software team members should be familiar with C/C++ and object-oriented programming.
+Experience with hardware interfaces such as I2C/SPI is a plus.
+
+The ground station and radio communication will require knowledge of physical radio properties and various radio modulation techniques.
+Other astronautical engineering skillsets may be required, dependent on unforseen technical challenges.
+
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+This project should not exceed \$500 in total costs incurred.
+
+The original \$50SAT project cost less than \$250 in raw materials so this project should be within a similar range.
+Potential sources for cost inflation include going with an external assembler of the PCBs, and unforseen changes in the design.
+
+An Interactive Learning Grant (ILG) should be able to cover the total expected cost of the project, however it would be prudent to have backup funding available in case of budget overruns.
+The project team should give the SPEX administration a preliminary BoM and cost estimate once their initial review of the designs are complete, to aid in securing funding.
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Faculty support is recommended for teams pursing this project.
+The electrical subsystems would most significantly benefit from faculty advisement to review the final assembled board and solar panel assemblies.
+Also the radio and antennas system testing would benefit from SPEX advisor expertise and oversight.
+
+\subsection{Long-Term Vision}
+\label{sec:vision}
+This project would result in SPEX's first functional, space-rated hardware project.
+
+Future teams could utilize the same low cost design philosphy while pushing the boundaries in one or more subsystems over time to create a more compelling spacecraft.
+The skils learned from this project would mitigate the risk of future projects with increased technical complexity.
+
+If this project was iterated on, hardware developments and experience gained from these projects could be applied to a CubeSat or other nanosatellite that would actually fly in space, through a launch partnership like CSLI.
+
+\section*{Acknowledgements}
+The author would like to thank Professor Bob Twiggs at Morehead State University and the rest of the \$50SAT team for publishing open source design for the mission, allowing similar student teams to work on flight-proven hardware.
+
```

Comment:
Latest commit has the new bibliography line, but it is not showing up. I think I removed the inline references in a prior version.